stan/math/fwd/scal/fun/log_inv_logit_diff.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef STAN_MATH_FWD_SCAL_FUN_LOG_INV_LOGIT_DIFF_HPP
+#define STAN_MATH_FWD_SCAL_FUN_LOG_INV_LOGIT_DIFF_HPP
+
+#include <stan/math/fwd/core.hpp>
+#include <stan/math/prim/scal/fun/log_inv_logit_diff.hpp>
+#include <stan/math/prim/scal/fun/inv_logit.hpp>
+#include <stan/math/prim/scal/fun/inv.hpp>
+#include <stan/math/prim/scal/fun/expm1.hpp>
+
+namespace stan {
+namespace math {
+
+using std::exp;
+
+template <typename T>
```

Comment:
These should all have doc as they're standalone functions.  It can be simple, but should indicate tparams, params, and return by name.